example/spec/loadTest/queueGranuleCompletedLoadTest.js

```diff
@@ -0,0 +1,277 @@
+const fs = require('fs');
```

Comment:
Is there a need for this test in the context of the ticket?   We should discuss and consider removing this for this PR unless it meets the specific objective of 2926.

Reply:
Removed queueGranuleCompletedLoadTest